lib/config.js

```diff
@@ -57,9 +57,10 @@ var statFromShell = process.env.STAT_HOST;
 var oauthFromShell = process.env.OAUTH_HOST;
 
 var envFromBrowser = locationMatch(/\W?env=(\w+)/);
+const envFromCreateReactApp = process.env.REACT_APP_ENV
```

Comment:
This is using `const`, which doesn't follow convention (other variable declarations use the old school `var`) and will break web browser compatibility...

...if that web browser is [Internet Explorer 10.](https://caniuse.com/#feat=const)

Wait, what am I complaining about?

Reply:
Good point. Will make the change